react/Well/styles.styl

```diff
@@ -0,0 +1,13 @@
+@require 'settings/palette.styl'
+@require 'settings/breakpoints'
+@require 'tools/mixins'
+
+.well
+  margin rem(16 32)
+  background paleGrey
+  border rem(1) solid silver
+  padding rem(16)
```

Comment:
Should we / can we use utility classes in cozy-ui itself?

Reply:
Why not ?